src/components/algolia-search/components/panel.tsx

```diff
@@ -97,16 +97,7 @@ export default forwardRef(function Panel<THit extends Hit<unknown>>(
   /**
    * Ensures the body can't be scrolled while the search panel is open on mobile viewports.
    */
-  useEffect(() => {
-    const initialValue = document.body.style.overflow
-    if (!isDesktop) {
-      document.body.style.overflow = 'hidden'
-    }
-
-    return () => {
-      document.body.style.overflow = initialValue
-    }
-  }, [isDesktop])
+  useNoScrollBody(!isDesktop)
```

Comment:
🙌 props for finding places that do the same thing and backfilling the abstraction 💖

Reply:
That was very much @ashleemboyer's idea 🙇 👏

Reply:
TY ❤️